### PR TITLE
Kms v2 model

### PR DESCRIPTION
**Pull request by ShubhamChaturvedi7**
*Issue #, if available:*

*Description of changes:*
Adds kms v2 models and enables it for Python.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: TestModels/aws-sdks/kmsv2/Model/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+This is the [KMS smithy model](https://github.com/aws/aws-models/blob/a7e23cbfde9a5fc0b52c78ff969d72bfb8d8b6f8/kms/smithy/model.json),
+but with a couple adjustments:
+
+- The `ListKeys` operation is removed from the model, and from `com.amazon.kms#KeyManagementService`.
+  - This is because the Dafny compiler does not allow "Keys" as a constructor parameter name.
+    See <https://github.com/dafny-lang/dafny/issues/1587>.
+- The `ListKeysResponse` structure is removed from the model.
+  - This is for the same reason as removing `ListKeys`.
```
Comment (robin-aws) [inline TestModels/aws-sdks/kmsv2/Model/README.md]: Just FYI these aren't necessary any more, and the `kms-lite` model doesn't do this.

File: TestModels/aws-sdks/kmsv2/README.md
Comment (robin-aws) [inline TestModels/aws-sdks/kmsv2/README.md:1]: Mentioned what v2 means somewhere (same comment as on the DDB one)

**Review by robin-aws — CHANGES_REQUESTED** [review]

File: TestModels/aws-sdks/kmsv2/Model/README.md (hunk 1 above)
Comment (ShubhamChaturvedi7) [reply TestModels/aws-sdks/kmsv2/Model/README.md]: Removed lines 4 to 8

File: TestModels/aws-sdks/kmsv2/README.md
Comment (ShubhamChaturvedi7) [reply TestModels/aws-sdks/kmsv2/README.md:1]: Add a line mentioning this is smithy 2.0 model and should be compatible to v2/latest sdk.

**Review by robin-aws — APPROVED** [review]

**Pull request merged**